aten/src/ATen/function_wrapper.py

```diff
@@ -291,7 +291,7 @@ def __init__(self, reason):
     'THSTensor*':
         CodeTemplate(
             'checked_tensor_unwrap('
-            '${arg_name}.tref,"${arg_name}",${arg_pos},false, '
+            '${arg_name},"${arg_name}",${arg_pos},false, '
```

Comment:
I'm guess this is dead code, because we don't have `THSTensor` anymore.

Reply:
I removed the `'THSTensor*'` case from `CHECKED_CAST`.